raiden/ui/cli.py

```diff
@@ -493,3 +497,26 @@ def append_report(subject, data):
     else:
         print('[5/5] smoketest had errors, report was written to {}'.format(report_file))
         sys.exit(1)
+
+
+@run.command()
+@OPTION_ADDRESS
+@OPTION_DATADIR
+def removedb(address, datadir):
```

Comment:
Agreed. These make more sense as global options. This basically comes down to me not being familiar enough with click. Will fix.